### ISSUE TITLE
[Feature Request] Reader mode toggle?

### ISSUE THREAD

**Issue by Cveinnt** [ansh/bionic-reading#72]
As titled, do you have any plans to add a reader mode toggle? 
Thanks for making this extension!

**Comment by X140Yu** [comment]
Yes, there is a plan to make a separate reader mode like the reader view in Firefox

**Comment by ansh** [comment]
The current recommendation for a Reader mode is to install a separate extension to provide a reader mode and to use that with conjunction of Jiffy Reader.

**Comment by cptSwing** [comment]
> The current recommendation for a Reader mode is to install a separate extension to provide a reader mode and to use that with conjunction of Jiffy Reader.

Do you have any suggestions for extensions on hand?

I've tried the following, none of which worked as intended:

[Reader View](https://addons.mozilla.org/en-US/firefox/addon/reader-viewer):
Seems similar to FF's built in Reader View (icon in location bar), page is loaded through extension, Jiffy can't find page.

[Readability based Reader View](https://addons.mozilla.org/en-US/firefox/addon/readability-based-reader-view):
Page is loaded through extension, Jiffy can't find page.

[Postlight Reader](https://addons.mozilla.org/en-US/firefox/addon/postlight-reader):
Page is **not** loaded through extension, Jiffy seems to find stuff to parse - but does nothing.

[Tranquility Reader](https://addons.mozilla.org/en-US/firefox/addon/tranquility-1/):
Page is **not** loaded through extension, Jiffy seems to find stuff to parse - but does nothing.

Cheers for any suggestions :)
